tests/Selector/NamedSelectorTest.php

```diff
@@ -54,6 +54,7 @@ public function testSelectors($fixtureFile, $selector, $locator, $expectedExactC
 
     /**
      * @dataProvider getSelectorTests
+     * @group legacy
```

Comment:
this tells the PHPUnit bridge that it is OK to use deprecated APIs in this test, because we are precisely testing the legacy API to ensure that we don't break the BC layer.